proxy-init/iptables/iptables.go

```diff
@@ -73,12 +73,15 @@ func formatComment(text string) string {
 
 func addOutgoingTrafficRules(commands []*exec.Cmd, firewallConfiguration FirewallConfiguration) []*exec.Cmd {
 	outputChainName := "PROXY_INIT_OUTPUT"
+	redirectChainName := "PROXY_INIT_REDIRECT"
 	executeCommand(firewallConfiguration, makeFlushChain(outputChainName))
 	executeCommand(firewallConfiguration, makeDeleteChain(outputChainName))
 
 	commands = append(commands, makeCreateNewChain(outputChainName, "redirect-common-chain"))
 
-	// Ingore traffic from the proxy
+	// Redirect calls originating from to the proxy
+	commands = append(commands, makeRedirectChainForOutgoingTraffic(outputChainName, redirectChainName, "redirect-non-loopback-local-traffic"))
```

Comment:
Since this rule is tied to the proxy UID, we should handle it similarly to the `makeIgnoreUserId` rule on line 87. I'd move this rule to line 86, and then pass `firewallConfiguration.ProxyUid` as a param to `makeRedirectChainForOutgoingTraffic`.